fs/config/obscure/obscure.go

```diff
@@ -47,6 +48,9 @@ func crypt(out, in, iv []byte) error {
 // This is done by encrypting with AES-CTR
 func Obscure(x string) (string, error) {
 	plaintext := []byte(x)
+	if math.MaxInt32-aes.BlockSize < len(plaintext) {
```

Comment:
I think if someone has put 4 GiB of text in a string we are obscuring then perhaps they deserve rclone to blow up! The fix is OK though.